### PR TITLE
feat(secrets-manager): exclude characters for password rotation applications

### PR DESCRIPTION
**Pull request by ahammond**
Fixes EXCLUDED_CHARACTERS part of #4144

### CODE REVIEW

File: packages/@aws-cdk/aws-secretsmanager/lib/secret-rotation.ts [hunk 1]
```diff
@@ -193,6 +193,13 @@ export interface SecretRotationProps {
    * @default - a new security group is created
    */
   readonly securityGroup?: ec2.ISecurityGroup;
+
+  /**
+   * Characters which should not appear in the generated password
+   *
+   * @default - do not pass any additional configuration to the Lambda rotation function
```
Comment (njlynch) [inline packages/@aws-cdk/aws-secretsmanager/lib/secret-rotation.ts:200]:
```suggestion
   * @default - no additional characters are explicitly excluded
```

**Review by njlynch — DISMISSED** [review]
Thanks for this PR! Can you include a test to verify the new functionality?

**Comment by ahammond** [comment]
@njlynch  I've added the UTs and made the wording change you suggested. Thank you!

**Review by njlynch — APPROVED** [review]
Looks great!

**Review by njlynch — APPROVED** [review]
Re-approving.

**Pull request merged**